src/components/Launch.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Image } from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet'
+
+const Launch = () => {
+  return (
+    <Image source={require('../../images/fulllaunch.png')} style={styles.container} />
+  )
+}
+
+const styles = EStyleSheet.create({
+  container: {
+    flex: 1,
+    width: null,
+    height: null,
```

Comment:
Nah, I think this is fine. It seems a bit off from working on PFE so much, but this makes sense if it's particular to RN.